src/components/modals/ModalComponent.tsx

```diff
@@ -66,9 +67,12 @@ class ModalComponent extends React.Component<ModalProps> {
     window.addEventListener('keydown', this.onKeyDown, true)
   }
 
-  close = () => {
+  close = (fullReload: any = false) => {
     this.animateAndClose!()
     this.props.onClose?.()
+    if (Capacitor.isNativePlatform() && fullReload) {
+      window.location.reload()
+    }
```

Comment:
`fullReload` can be removed if reloading the page is no longer necessary.

Reply:
It seems we _can_ remove this. I need to look into why it's no longer necessary, but it seems to be working fine without the reload.

Removed: 4792aeb5696e436ff39808f46a8e7ecd6ad16d86